spec/game.test.js

```diff
@@ -0,0 +1,110 @@
+var Game = require('../public/src/game')
+
+describe('Game class', function () {
+  describe('#roll', function () {
+    var game
+
+    beforeEach(function () {
+      game = new Game()
+    })
+
+    it('Makes a new frame for new Game', function () {
+      game.roll(4)
+      expect(game.frames.length).toEqual(1)
+    })
+
+    it('Makes a new frame for 3rd roll', function () {
+      game.roll(3)
+      game.roll(3)
+      game.roll(3)
+      expect(game.frames.length).toEqual(2)
+    })
+
+    it('sends score to all frames', function () {
+      game.roll(3)
+      var firstFrame = game.frames[0]
+      expect(firstFrame.calculateTotalScore()).toEqual(3)
+    })
+
+    it('stops making new frames after 10 frames', function () {
+      for (var i = 0; i < 25; i++) {
+        game.roll(2)
+      }
+      expect(game.calculateTotalScore()).toEqual(40)
+    })
+
+    it('raises error for non-numeric inputs', function () {
+      expect(function () { game.roll('not_a_number') }).toThrow('Not a Number')
+    })
+
+    it('raises an error if the input over 10', function () {
+      var pinsSpy = {
+        isImpossibleScore: function () {
+          throw new Error('Not enough pins')
+        }
+      }
+      var args = { pins: pinsSpy }
+      var game = new Game(args)
+      expect(function () { game.roll(12) }).toThrow('Not enough pins')
+    })
+  })
+
+  describe('#calculateTotalScore', function () {
+    var game
+
+    beforeEach(function () {
+      game = new Game()
+    })
+    it('sums the score of all frames', function () {
+      game.roll(3)
+      game.roll(3)
+      game.roll(3)
+      expect(game.calculateTotalScore()).toEqual(9)
+    })
+
+    it('works for spares', function () {
```

Comment:
Try and be a bit more descriptive with your tests. Remember that part of the purpose of tests is to serve as documentation for the intent of the code. Saying something 'works' isn't as descriptive as describing what you actually want it to achieve, e.g. in this case 'applies a bonus on the next roll when the frame is a spare'.